index.html

```diff
@@ -15,9 +15,9 @@
             <a href="#" class="logo">Tshepo45</a>
              
             <ul class ="navMenu" >
-              <li class="nav-item"><a href="portfolio.html"class="nav-link">Portfolio</a></li>
-              <li class="nav-item"><a href="about"class="nav-link">About</a></li>
-              <li class="nav-item"><a href="Contact" class="nav-link" >Contact</a></li>
+              <li class="nav-item"><a href="#portfolio"class="nav-link">Portfolio</a></li>
+              <li class="nav-item"><a href="#about"class="nav-link">About</a></li>
+              <li class="nav-item"><a href="#Contact" class="nav-link" >Contact</a></li>
```

Comment:
- [ ] it would be better if you navigation links displayed on the desktop version right now they aren't so please try to fix that 
- Yours 
![Screenshot 2023-03-14 at 11 40 26](https://user-images.githubusercontent.com/61790587/224976089-a56b1a6b-d393-48b3-a96f-e5212b2f6071.png)

- Design 
![Screenshot 2023-03-14 at 11 40 20](https://user-images.githubusercontent.com/61790587/224976118-7404e3bb-a557-4c81-bf52-ddc618e0bb56.png)